interactions/api/models/guild.py

```diff
@@ -1546,6 +1546,42 @@ async def delete_emoji(
             reason=reason,
         )
 
+    async def get_list_of_members(
+        self,
+        limit: Optional[int] = 1,
+        after: Optional[int] = None,
+    ) -> list:
+        """
+        Lists the members of a guild.
+
+        :param limit?: How many members to get from the API. Max is 1000.
+        :type limit: Optional[int]
+        :param after?: Get Member IDs after this snowflake.
+        :type after: Optional[int]
```

Comment:
Add `:return:` and `:rtype:`

Reply:
doc: add docstring for returning values in get_list_of_members

Reply:
bruh

Reply:
fixed